pkgs/development/tools/wrangler/default.nix

```diff
@@ -13,16 +13,18 @@ rustPlatform.buildRustPackage rec {
 
   cargoSha256 = "0w845virvw7mvibc76ar2hbffhfzj2v8v1xkrsssrgzyaryb48jk";
 
-  nativeBuildInputs = [ perl ] ++ lib.optionals stdenv.isLinux [ pkg-config ];
+  nativeBuildInputs = [ pkg-config ];
 
-  buildInputs = lib.optionals stdenv.isLinux [ openssl ]
+  buildInputs = [ openssl ]
```

Comment:
This change with `OPENSSL_NO_VENDOR` causes the build to link against both `openssl` and `Security` on darwin?

Reply:
```
otool -L /nix/store/s3andqmkcm1f2ia7z98qrf3x103cxay1-wrangler-1.13.0/bin/wrangler
/nix/store/s3andqmkcm1f2ia7z98qrf3x103cxay1-wrangler-1.13.0/bin/wrangler:
        /System/Library/Frameworks/CoreServices.framework/Versions/A/CoreServices (compatibility version 1.0.0, current version 775.19.0)
        /nix/store/fazy81w6z16xsdcpyp44gkvf62l5wgn4-curl-7.74.0/lib/libcurl.4.dylib (compatibility version 12.0.0, current version 12.0.0)
        /nix/store/xxmdw2z5cm1rlcl8n9n91q7356542byw-openssl-1.1.1j/lib/libssl.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
        /nix/store/xxmdw2z5cm1rlcl8n9n91q7356542byw-openssl-1.1.1j/lib/libcrypto.1.1.dylib (compatibility version 1.1.0, current version 1.1.0)
        /usr/lib/libSystem.B.dylib (compatibility version 1.0.0, current version 1238.60.2)
        /System/Library/Frameworks/Security.framework/Versions/A/Security (compatibility version 1.0.0, current version 57740.60.18)
        /System/Library/Frameworks/CoreFoundation.framework/Versions/A/CoreFoundation (compatibility version 150.0.0, current version 1349.8.0)
        /nix/store/v5vd9k28hsmj826nr0sx48ajyh7wfqf1-Libsystem-1238.60.2/lib/libresolv.9.dylib (compatibility version 1.0.0, current version 1.0.0)
```

Reply:
> This change with `OPENSSL_NO_VENDOR` causes the build to link against both `openssl` and `Security` on darwin, was it using the vendored `openssl` previously?

Do you mean on the Mac? I don't know for sure, I don't have a Mac.

Reply:
> Do you mean on the Mac? I don't know for sure, I don't have a Mac.

Oops, for some reason I thought you did. My bad.

Just checked and it was using the vendored `openssl` on darwin.

Reply:
I used to have a Mac, but recently replaced my last MacBook by a ThinkPad.

Thanks for checking!